.travis.yml

```diff
@@ -16,6 +16,10 @@ cache:
   directories:
     - js/node_modules
 
+
+before_install:
+  - export COCOAPODS_FIX_REACT_NATIVE_DEV_ROOT=subtrees/the-blue-alliance-react/node_modules/react-native
```

Comment:
We might look to move this someplace in our `Fastfile` so we can get this functionality locally as well